eng/testing/RunnerTemplate.sh

```diff
@@ -155,6 +103,10 @@ elif [[ "$(uname -s)" == "Linux" ]]; then
 
   ulimit -c unlimited
 fi
+
```

Comment:
@hoyosjs - is above core file configuration for `/proc/self/coredump_filter` still necessary?  Wat about `ulimit` settings?

Reply:
If the helix machines already have this configuration enabled, I'll remove both things.